src/storage/sqlite.py

```
@@ -4059,21 +4156,61 @@ def latest_regime_snapshot(self) -> RegimeSnapshotRecord | None:
                 """,
                 params,
             ).fetchone()
-        return self._row_to_event(row) if row is not None else None
+        return self._row_to_econ_event(row) if row is not None else None
+
+    def _row_to_econ_event(self, row: sqlite3.Row) -> StoredEventRecord:
+        parsed_event_time = datetime.fromisoformat(
+            str(row["event_time_utc"]).replace("Z", "+00:00")
+        )
+        if parsed_event_time.tzinfo is None:
+            parsed_event_time = parsed_event_time.replace(tzinfo=timezone.utc)
```

Ruff (0.15.10)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)